openbb_platform/providers/fmp/openbb_fmp/models/earnings_call_transcript.py

```diff
@@ -23,7 +23,8 @@ class FMPEarningsCallTranscriptQueryParams(EarningsCallTranscriptQueryParams):
     def time_validate(cls, v: int):  # pylint: disable=E0213
         """Return the year as an integer."""
         current_year = datetime.now().year
-        return current_year if v > current_year or v < 1950 else v
+        # Current year + 1 to account for potential fiscal year.
+        return current_year if v > (current_year + 1) or v < 1950 else v
```

Comment:
Let's just remove this validator entirely. It does not need to exist

![Screenshot 2024-06-13 at 4 07 44 PM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/ee0f07cf-0ff8-4179-9507-a26d4bd3705c)